parsers/iniconf.c

```diff
@@ -34,8 +34,9 @@
 
 static bool isIdentifier (int c)
 {
-    /* allow whitespace within keys and sections */
-    return (bool)(isalnum (c) || isspace (c) || c == '_' || c == '-' || c == '.');
+	/* allow whitespace within keys and sections */
+	return (bool)(isalnum (c) || isspace (c) || c == '_' || c == '-' ||
+		c == '.' || c == '"' || c == '\'');  /* for TOML */
```

Comment:
I want you to be clear the range of text that `/* for TOML */` qualifies more.
What do you think about using macro like:
```C
#define TOML_ID_CHAR(C) ( (c) == '"' || (c) == '\'' )
return (bool)(isalnum (c) || isspace (c) || c == '_' || c == '-' || c == '.'
    || (TOML_ID_CHAR(c));
```

Reply:
Done.